task.go

```diff
@@ -8,6 +12,6 @@ type Task struct {
 	// The weight is used to distribute goroutines over multiple tasks.
 	Weight int
 	// Fn is called by the goroutines allocated to this task, in a loop.
-	Fn   func()
+	Fn   func(args TaskArgs)
```

Comment:
Because boomer has not verisons now, I'm not prefer to add args to Fn, which will break many existed builds. Instead, I'm planning to add an API, which allows tasks to have access to remote params. See also https://github.com/myzhan/locust4j/commit/9679d7f152a12667c2da75628ef61b7410d09e84